scripts/benchmarks/main.py

```diff
@@ -71,28 +115,31 @@ def main(directory, additional_env_vars, save_name, compare_names, filter):
                 print(f"running {benchmark.name()}, iteration {iter}... ", end='', flush=True)
                 bench_results = benchmark.run(merged_env_vars)
                 if bench_results is not None:
-                    print(f"complete ({bench_results.value} {benchmark.unit()}).")
-                    iteration_results.append(bench_results)
+                    for bench_result in bench_results:
+                        print(f"complete ({bench_result.label}: {bench_result.value} {benchmark.unit()}).")
+                        iteration_results.append(bench_result)
                 else:
                     print(f"did not finish.")
 
             if len(iteration_results) == 0:
                 continue
 
-            iteration_results.sort(key=lambda res: res.value)
-            median_index = len(iteration_results) // 2
-            median_result = iteration_results[median_index]
+            for label in set([result.label for result in iteration_results]):
+                label_results = [result for result in iteration_results if result.label == label]
+                label_results.sort(key=lambda res: res.value)
+                median_index = len(label_results) // 2
+                median_result = label_results[median_index]
 
-            median_result.unit = benchmark.unit()
-            median_result.name = benchmark.name()
+                median_result.unit = benchmark.unit()
+                median_result.name = label
```

Comment:
Why did you change this? Now, we have 'hashtable' instead of 'VelocityBench Hashtable'